### PR TITLE
A11Y-7: Add alt text to careers page images

### PR DESCRIPTION
**Pull request by ebeastlake**
This PR adds alt text for these two images. I decided to give the all-employee photo empty alt text because it doesn't feel like it conveys anything in particular, but the Code.org pride photo feels worth explaining.

Screen Shot 2022-12-14 at 10 29 36 AM
 
## Links

Jira ticket: https://codedotorg.atlassian.net/browse/A11Y-7

## Testing story

Tested locally with VoiceOver (see output below).

Screen Shot 2022-12-14 at 10 27 13 AM

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -7255,6 +7255,8 @@
 
   language_label: "Language"
 
+  careers_page_code_pride_alt_text: "Two female-presenting employees smile while holding the ends of a Code.org-branded rainbow banner. Behind them, several people are wearing Code.org shirts and waving rainbow flags while walking alongside a multi-colored Volkswagon bug."
```
Comment (molly-moen) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: typo: should be "Volkswagen"

**Review by molly-moen — APPROVED** [review]
LGTM, just one typo :)

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 1 above)
Comment (megcrenshaw) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: I really like that you added this description!

Nit: I think `female` has a biological connotation that I don't think you're intending. Maybe `feminine-presenting` or `woman-presenting`? Language for gender is evolving which makes this hard!

Comment (megcrenshaw) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Wow TIL

Comment (ebeastlake) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Good catch!

**Review by hannahbergam — APPROVED** [review]
Heck yes! Thank you for incorporating feedback and writing a great description!

**Pull request merged**